code/datums/sexcon/sexcon.dm

```diff
@@ -257,10 +257,10 @@
 				cameloc = target.getorganslot(ORGAN_SLOT_BREASTS)
 			if(vaginal || anal || nipple)
 				var/cum_to_take = CLAMP((testes.reagents.maximum_volume/4), 1, min(testes.reagents.total_volume, cameloc.reagents.maximum_volume - cameloc.reagents.total_volume))
-				testes.reagents.trans_to(cameloc, cum_to_take, transfered_by = user)
+				testes.reagents.trans_to(cameloc, cum_to_take, transfered_by = user, method = INGEST)
```

Comment:
Does that make sense to use ingest while transfering into a container that is the organ?

Reply:
Felt it was the most fitting in this case, either TOUCH, INGEST, VAPOR, PATCH or INJECT. Inject just sounded too clinical to me. Not that it truly matters, it is just there so the variable is set. It is not used.

It just needed to be set to pass here:
https://github.com/StoneHedgeSS13/StoneHedge/blob/2e31403ca9211385f5bdef241f3a15e5f0e004dd/code/modules/reagents/chemistry/holder.dm#L212